components/clm/bld/namelist_files/use_cases/2010_CMIP6_control.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+<use_case_desc>Conditions to simulate 2010 land-use</use_case_desc>
+
+<sim_year>2015</sim_year>
```

Comment:
Not 2010?

Reply:
This is because land only supports certain years. Unfortunately, 2015 is not one of them. Same adjustment is also used for other compset.

Reply:
Thanks for the reply.